util/journaldb/src/lib.rs

```diff
@@ -16,51 +16,86 @@
 
 //! `JournalDB` interface and implementation.
 
-extern crate parity_util_mem;
-extern crate parity_util_mem as mem;
 extern crate parity_util_mem as malloc_size_of;
```

Comment:
Have you considered renaming it in `Cargo.toml`? (https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#renaming-dependencies-in-cargotoml)

Reply:
oh, I didn't know it's possible! thanks :)